cns/restserver/api.go

```diff
@@ -1231,7 +1235,7 @@ func (service *HTTPRestService) publishNetworkContainer(w http.ResponseWriter, r
 
 		if isNetworkJoined {
 			// Publish Network Container
-			returnMessage, returnCode = service.doPublish(ctx, req, ncParameters)
+			returnMessage, returnCode, publishStatusCode = service.doPublish(ctx, req, ncParameters)
```

Comment:
This pattern also needs to be reflected in the unpublish code. DNC does similar checks on the response.

Reply:
Addressed: [620dd55](https://github.com/Azure/azure-container-networking/pull/1799/commits/620dd55a161595ff01da33f08964a273339043de)